app/design/frontend/Elogic/maksym/Magento_Catalog/web/css/source/_extend.less

```diff
@@ -0,0 +1,7 @@
+.page-with-filter {
```

Comment:
@imports should be outside of a selector

Reply:
done